pytorch_lightning/loggers/tensorboard.py

```diff
@@ -57,6 +57,9 @@ class TensorBoardLogger(LightningLoggerBase):
             directory for existing versions, then automatically assigns the next available version.
             If it is a string then it is used as the run-specific subdirectory name,
             otherwise ``'version_${version}'`` is used.
+        sub_dir: Sub-directory to group TensorBoard logs. If a sub_dir argument is passed
+            then logs are saved in ``/save_dir/version/``. Defaults to ``None`` in which
```

Comment:
```suggestion
            then logs are saved in ``/save_dir/version/sub_dir/``. Defaults to ``None`` in which
```

At least that's what I assume, since otherwise this would be pointless :)

Reply:
Ah my bad, thanks for spotting this!